app/models/mail_server_log/translated_constants.rb

```diff
@@ -0,0 +1,16 @@
+# -*- encoding : utf-8 -*-
+class MailServerLog::DeliveryStatus
```

Comment:
The path to this file should be `app/models/mail_server_log/delivery_status/translated_constants.rb`